src/DayColumn.jsx

```diff
@@ -159,7 +159,7 @@ let DaySlot = React.createClass({
           key={'evt_' + idx}
           style={{...xStyle, ...style}}
           title={label + ': ' + title }
-          onClick={this._select.bind(null, event)}
```

Comment:
So the extra parameters being passed were due to using `.bind` instead of using an arrow function to explicitly pass the `SyntheticEvent` up the chain. Apparently, React's event handler signature is actually `function (SyntheticEvent, null, SyntheticEvent) {}`. By using `.bind` it was passing the additional parameters as well. 

This looks like a small bug in React that will be fixed in a future release (and really has no impact here other "code-smell"). You can follow more on the topic [here](https://github.com/facebook/react/issues/7902).

Anyway, it is fixed now. I'd appreciate if you could confirm that the existing binding with no context was not implemented for a specific reason. @jquense you wrote both lines of code within 2 days of each other so was there a change of heart on binding vs arrow functions? Or just random coding decision? Or another reason?